src/bindings/python/requirements_test.txt

```diff
@@ -2,7 +2,7 @@
 numpy
 bandit
 black
-flake8
+flake8<=7.0.0
```

Comment:
do we know whether other `flake8-*` packages should be limited?

Reply:
As far as I know they are independent addons to `flake8`, so yes. We've recently had an issue on a release branch where `flake8-pytest-style` got updated independently of `flake8`.

Reply:
added upperbounds to all flake8 checks